app/styles/styles.css

```diff
@@ -18,6 +18,10 @@
   color: #ffffff;
 }
 
+.todo-app-component input {
+  font-family: 'Open Sans', sans-serif;
```

Comment:
normalize.css (@imported) specifies rules on form tags (button, input, optgroup, select, textarea). Those rules override any font family specified on the html tag. Maybe a more general way to set the font-family would be to set it on all the html and form tags.